### DISCUSSION TITLE
NocoDB on Supabase [Jun 02]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2313] [Discord Threads]
> 🙋‍♂️ **Gilbert**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### Since we connect to any underlying db and supabase is postgres. So the answer is yes.  
#  
> 💁 **o1lab**  
#### multitenant is not yet possible  
#  
> 💁 **o1lab**  
#### if you can write more on the requirement that would be helpful  
#  
> 🙋‍♂️ **Gilbert**  
#### I work for a company that has a messaging platform + chatbot automation. Our bot flows can interact with data from third-party systems via APIs   
####    
#### Many of our clients like to see their data in a spreadsheet format, such as Google Sheets. However, when the number of rows grows the response time grows as well. So, I want to explore the possibility to build a backend using NocoDB, and Supabase as the DB and deploying it on Netlify. Besides having some end-user views in NocoDB and would like to be able to build other kinds of reports on the front end.   
####    
#### I was asking about multitenant capabilities since we need to support many different users.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/981306236576342046)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
